### ISSUE TITLE
Linear Interpolation

### ISSUE THREAD

**Issue by affromero** [HsinYingLee/DRIT#22]
Can you share please how you do the linear interpolation between two attributes? Is it just a linspace?
Thank you.

**Comment by hytseng0509** [comment]
Yes, I believe you can use a linspace for each dimension of attributes to do the interpolation.
